content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -5,53 +5,139 @@ weight = 50
 
 Referenced from [Workload Identity Documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity). Please read this doc for:
 
-* Detailed introduction about workload identity.
+* A detailed introduction to Workload Identity.
 * Instructions to enable it on your cluster.
 * Whether its limitations affect your adoption.
 
 #### Terminology
 
 This document distinguishes between [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) (KSAs) and [Google service accounts](https://cloud.google.com/iam/docs/service-accounts) (GSAs). KSAs are Kubernetes resources, while GSAs are specific to Google Cloud. Other documentation usually refers to both of them as just "service accounts".
 
-#### Authoring pipelines to use workload identity
+#### Authoring pipelines to use Workload Identity
 
-Pipelines don't need any change to authenticate to GCP, it will use the GSA bound to `pipeline-runner` KSA transparently.
+Pipelines don't need any specific changes to authenticate to Google Cloud. With Workload Identity, pipelines run as the Google service account that is bound to the KSA.
 
-However, existing pipelines that use [use_gcp_secret kfp sdk operator](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.extensions.html#kfp.gcp.use_gcp_secret) need to remove `use_gcp_secret` usage to use the bound GSA.
-You can also continue to use `use_gcp_secret` in a cluster with workload identity enabled, but pipeline steps with `use_gcp_secret` will use the GSA corresponding to the secret provided.
+However, existing pipelines that use [use_gcp_secret kfp sdk operator](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.extensions.html#kfp.gcp.use_gcp_secret) need to remove the `use_gcp_secret` usage to use the bound GSA.
+You can also continue to use `use_gcp_secret` in a cluster with Workload Identity enabled, but pipeline steps with `use_gcp_secret` will use the GSA corresponding to the secret provided.
 
-#### Cluster setup to use workload identity for Pipelines Standalone
+#### Cluster setup to use Workload Identity for Pipelines Standalone or AI Platform Pipelines
 
-**After you enabled workload identity**, you need to bind workload identities for KSAs used by [Pipelines Standalone](/docs/pipelines/installation/overview/#kubeflow-pipelines-standalone).
-The following helper bash scripts bind workload identities for KSAs provided by Pipelines Standalone:
+##### 1. Create your cluster with Workload Identity enabled
```

Comment:
Would you rename "Pipelines Standalone" as " Pipelines Standalone CLI"? AI Platform Pipelines also is standalone.

Reply:
AI Platform Pipelines UI accordinglly.

Reply:
I prefer you remove "or AI Platform Pipelines" for now before we well support Workload Identity. It's not fully tested and prefer let UI installed cluster be managed by MKP stye.

Later in Q2/Q3, after support WI in AI Platform Pipelines UI/UI-based CLI, then we put it back.

Reply:
We may confirm with @paveldournov, so far, the branding of AI Platform Pipelines have nothing to do with Standalone. We may change that later after discussion.